packages/mwp-tracking-plugin/src/util/trackingUtils.js

```diff
@@ -52,3 +57,10 @@ export const parseIdCookie = (
 	}
 	return parsed.id;
 };
+
+// ZoneId.toString() adds the timezone in brackets, which we want to remove
+// e.g.: 2019-01-07T10:27:02.791-05:00[America/New_York]
+export const getZonedDateTimeStringWithUTCOffset = () =>
+	ZonedDateTime.now(ZoneId.of('America/New_York'))
+		.toString()
+		.replace(/\[.*\]$/, '');
```

Comment:
not a blocker, but a slightly more efficient regex would be

```js
/\[[^\]+]\]$/
```

because the parser doesn't have to backtrack, but it's a bit harder to read so whatevs